dask_cuda/cli/dask_cuda_worker.py

```diff
@@ -112,6 +112,15 @@
     "WARNING: managed memory is currently incompatible with NVLink, "
     "trying to enable both will result in an exception.",
 )
+@click.option(
+    "--rmm-log-directory",
+    default=None,
+    help="Directory to write per-worker RMM log files to. If users "
+    "want logging from the client or scheduler, that must be set "
+    "separately."
+    "NOTE: Has no effect if --rmm-pool-size is not specified and "
+    "--rmm-managed-memory is not enabled.",
```

Comment:
I don't think that's true, RMM logs are valid independent of the underlying memory management resource underneath.

Reply:
Should we allow RMM logging regardless of those decisions then? Right now, logging is enabled through `RMMSetup`, and only if `self.nbytes is not None or self.managed_memory is True`.

Reply:
Ah ok, I see what you're referring to. The sentence then may be a bit misleading to dumber people like me (unclear if "no" or "and" has precedence). I would rephrase it to:
"NOTE: Logging will only be enabled if --rmm-pool-size or --rmm-managed-memory are specified."

Reply:
Okay - out of interest, is there any utility to having logging on if we don't use a pool or managed memory?

Reply:
Theoretically, yes. Libraries such as cuDF use RMM by default, even if it doesn't use a pool or managed memory. However, we generally want people to use a pool, so I'm not sure whether logging for all cases is something we want to incentivize. I'm ok either way, but my personal opinion is we don't need it now.